### ISSUE TITLE
Colab Pro free for students

### ISSUE THREAD

**Issue by karkir0003** [googlecolab/colabtools#3325]
It would be great if google colab could give colab pro free for university students. It would be extremely helpful if colab pro could be added as part of the Github student developer pack so that we can better democratize access to GPU, TPU technologies for more people to dive into ML/DL and get their hands dirty with big data with minimal setup :)

**Comment by karkir0003** [comment]
@colaboratory-team @fischman @eric-wieser @craigcitro @yilei

**Comment by karkir0003** [comment]
what's the link to file in-product feedback? @EvanWiederspan

**Comment by YuenSzeHong** [comment]
same question/request

**Comment by Endymi0nn** [comment]
Yes please same request from me

**Comment by AzzedineAftiss** [comment]
yes please

**Comment by aidyai** [comment]
Yes Please, it will be be a huge boost

**Comment by shashankbl** [comment]
@EvanWiederspan Colab Pro+ at free/heavily discounted option would be great for grad students! Please consider it :)

**Comment by KauanNovello1106** [comment]
It would be great, I really need it for a data analysis project at college but my machine doesn't support that much RAM memory

**Comment by yukiarimo** [comment]
+

**Comment by dpgb4o** [comment]
Agree!!!!!

**Comment by Cioraz** [comment]
100% agree on this

**Comment by kenza-benkirane** [comment]
Agree!

**Comment by JohannFaust666** [comment]
I need it so bad

**Comment by Tanujshriyan** [comment]
#2367

**Comment by lxyeternal** [comment]
I agree!!!

**Comment by D3lik** [comment]
I agree, it will be much more helpful to students all around the world

**Comment by vitos-exe** [comment]
Agree

**Comment by AryanMadhu** [comment]
Agree, Please make free for College Students, I need it for ML/DL/RL Projects

**Comment by MoAbbasid** [comment]
I also agree, as student our budgets are constrained and we dont get that many resources, Microsoft is already offering some free benefit for students.
Colab Pro for student would be of great help to us.

**Comment by taneya1987** [comment]
+1
I can definitely make use of it today. Being a university student and that too doing Master in Artificial Intelligence, this is something which we should be provided with.

**Comment by karkir0003** [comment]
> It would be great if google colab could give colab pro free for university students. It would be extremely helpful if colab pro could be added as part of the Github student developer pack so that we can better democratize access to GPU, TPU technologies for more people to dive into ML/DL and get their hands dirty with big data with minimal setup :)

Anyway we can get Google to review this feature request. I tried submitting this request via the colab page, but not sure if it got received by the team

**Comment by boyin96** [comment]
Agree!

**Comment by JardelCunha** [comment]
some news about that? same question from me

**Comment by dehlong** [comment]
+1

**Comment by karkir0003** [comment]
> Thanks, this is something we're definitely interested in for the future. If you file in-product feedback, its easier for us to aggregate requests for this together.

I filed in product feedback within Colab. Could we have the team look at it? @EvanWiederspan

**Comment by edofiore** [comment]
+1 would be amazing

**Comment by Jameson-Crate** [comment]
+1 this would be great

**Comment by Monsurat-Onabajo** [comment]
+1 this would be great to have, where do we file product feedback

**Comment by MohsinRaz3** [comment]
As students on a budget with limited resources, Colab Pro would really help us out

**Comment by jpaanane** [comment]
Agree

**Comment by GrenRadon** [comment]
Totally agree from Colombia

**Comment by arko-14** [comment]
surely,this would help the student community very much.

**Comment by yanyunn** [comment]
Agree, we really need these resources.

**Comment by Monsurat-Onabajo** [comment]
Hello everyone, you can checkout kaggle, you can use gpt and tpu for 30 hours weekly after, it helped me in my project

**Comment by pesawaterbang** [comment]
I am research student, serious google colab more impactful for GPU.
but please pro discount for student.

**Comment by flashdash101** [comment]
Doesn't even have to be free, would love a discount!

**Comment by ddp22** [comment]
Totally agree

**Comment by dm-shr** [comment]
Even a discount would be very helpful for me and others!

**Comment by Monsurat-Onabajo** [comment]
Hello everyone, you can check out kaggle notebook...they give free GPU and
TPU up to 30 hours weekly, just ensure it is switched off whenever you are
not using it so it doesn't count because every hour it is switched on
counts.

Merry Christmas to you all

**Comment by sirine707** [comment]
Dear Github Support Team,
I have the same request , as a masters student Colab Pro will be helpful in this era and will definitely leverages my skills to contribute to ML projects .
Thank you for all your efforts .
Kind Regards.

**Comment by Abib-web** [comment]
Hi teams,
I thinks thats good idea

**Comment by ovindu17** [comment]
agree!!

**Comment by Prokilo1987** [comment]
> It would be great if google colab could give colab pro free for university students. It would be extremely helpful if colab pro could be added as part of the Github student developer pack so that we can better democratize access to GPU, TPU technologies for more people to dive into ML/DL and get their hands dirty with big data with minimal setup :)

Absolutely agree, I hope they do

**Comment by Online13** [comment]
Agree !

**Comment by SakshamKapoor2911** [comment]
Agree!

**Comment by shadowhexer** [comment]
Same

**Comment by raisedin-carolina** [comment]
agree

**Comment by axiom-guy** [comment]
agree

**Comment by Chamal1120** [comment]
agree

**Comment by DNNYF** [comment]
I am also experiencing limited computer resources as a student. So I agree 👍

**Comment by SakshamKapoor2911** [comment]
Thank you so much! We finally got free Colab Pro for students for 12 months!

**Comment by karkir0003** [comment]
> Thank you so much! We finally got free Colab Pro for students for 12 months!

For real @SakshamKapoor2911 ? If so, hats off to the colab team considering the times.

**Comment by karkir0003** [comment]
> Thank you so much! We finally got free Colab Pro for students for 12 months!

Ah. I see it: https://blog.google/outreach-initiatives/education/colab-higher-education/

**Comment by dewangbhanushali-25** [comment]
its happened guys , it happened

**Comment by lokman2k5** [comment]
sadly it's not available in some countries

<img width="424" height="357" alt="Image" src="https://github.com/user-attachments/assets/13c61903-2443-40a4-9c3f-dfaa68ed9608" />

please make it available in more countries!

**Comment by judemars** [comment]
Hi from the Colab team! Yes, as folks have mentioned, we have released a limited number of no cost Pro subscriptions to eligible, US-based students and faculty. Read more in [our announcement blog](https://blog.google/outreach-initiatives/education/colab-higher-education/), signup [here](https://colab.research.google.com/signup), and [see our FAQ](https://research.google.com/colaboratory/faq.html#edu-faq-heading) for more technical questions!

**Comment by beenycool** [comment]
not for any other countries? like uk ect

**Comment by danishnaseer00** [comment]
yes please make it free for students
